yarn.lock

```diff
@@ -10,13 +10,20 @@
     "@jridgewell/gen-mapping" "^0.1.0"
     "@jridgewell/trace-mapping" "^0.3.9"
 
-"@aragon/osx-ethers@1.3.0-rc0", "@aragon/osx-ethers@^1.3.0-rc0":
+"@aragon/osx-ethers@1.3.0-rc0":
```

Comment:
Why do we have two versions here?

Reply:
> there is only one which I believe is the correct one

We now have 
`"@aragon/osx-ethers@1.3.0-rc0":`
and
`"@aragon/osx-ethers@^1.3.0-rc0.1":`
as dependencies.

Not saying this is important, but just makes me wonder why. Go ahead

Reply:
no, you are absolutely right, I forgot the update the osx-ethers version on the client package.json, thank you